parser/src/Tags.ts

```diff
@@ -266,6 +267,7 @@ export default function RemoteExtension(this: RemoteExtensionContext) {
     "end_targeted_widget_attribute",
     "end_widget_container",
     "end_widget_wrapper",
+    "end_json_block",
```

Comment:
Custom HubL tags get registered here. This is left over from before we completely forked the jinja parser and could eventually move into the parser itself but things work fine here for now